main.py

```diff
@@ -109,8 +114,13 @@ def _request_flow_list(self, switch):
                                                              switch)
             xid_ports = of_core_v0x04_utils.request_port_stats(self.controller,
                                                                switch)
-            self._multipart_replies_xids[switch.id] = {'flows': xid_flows,
-                                                       'ports': xid_ports}
+            xid_tables = of_core_v0x04_utils.update_table_list(self.controller,
```

Comment:
`MultipartType.OFPMP_TABLE` request should only be sent if the switch supports this [capability as mentioned in the blueprint](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP035.rst#of_core), otherwise we don't know what unexpected behavior or errors can happen, can you adapt the code accordingly?  `'tables'` key shouldn't be set for a `switch.id` if it hasn't been sent.

Reply:
@viniarck, Fine, I agree. but I have a question for 
' controller.switches[dpid].features.capabilities.value & Capabilities.OFPC_TABLE_STATS == Capabilities.OFPC_TABLE_STATS`.

 `switch.features.capabilities.value`is taken from switch, but `Capabilities.OFPC_TABLE_STATS`, where is it taken from?

Reply:
Thank you, @viniarck